tsconfig.json

```diff
@@ -16,10 +16,13 @@
     "noImplicitAny": false,
     "strictBindCallApply": false,
     "forceConsistentCasingInFileNames": false,
-    "noFallthroughCasesInSwitch": false
+    "noFallthroughCasesInSwitch": false,
+    "paths": {
+      "@/*": ["src/*"],
```

Comment:
What do you think about adding more informative aliases since this early stage of the project?
Like @modules/ @services/ @repositories/ etc

Reply:
You’re absolutely right. I will add more aliases in the next PR to avoid conflicts.